### PR TITLE
Code part

### PR DESCRIPTION
**Pull request by laurwang**
I went through the READMEs for Lessons 0 through 3 and made a few changes.  Please see the diffs below.  By the way, I usually use this at the command line to deploy instead of the version you use:
$ sls deploy -s BIMX -v
(not sure it's worth changing in the write-up).  Also, may need a bit of discussion on private.yml.  (I added just a line about it.)

### CODE REVIEW

File: Lesson0_BeforeWorkshop/README.md [hunk 1]
```diff
@@ -57,8 +58,9 @@ $ export AWS_ACCESS_KEY_ID=<access-key-id>
 $ export AWS_SECRET_ACCESS_KEY=<secret-access-key>
 $ export AWS_SESSION_TOKEN=<session-token>             # this one is optional
 ```
+###Step 4: serverless deployments require some information you may not want to check in to a public repo.  Fill in the information in private.yml.
 
-###Step 3: install serverless v1.16+ node package on your machine.
+###Step 5: install serverless v1.6 node package on your machine.  *For the moment, there are some issues with the latest Serverless package.  See* https://github.com/serverless/serverless/issues/3253.
```
Comment (gwsii) [inline Lesson0_BeforeWorkshop/README.md]: Do we need to make provisions for the fact that our current workshop code doesn't work with and official release of SLS? We've been using a private version from Eric's github account:  i.e.:   "serverless": "git+https://github.com/erikerikson/serverless.git#allow-self-reference",

**Comment by laurwang** [comment]
The workshop code does not rely on the self-referencing that needs Erik’s version.

**Comment by erikerikson** [comment]
is this PR superseded by #16 ?

**Comment by laurwang** [comment]
Very much so.  Closing.

**Pull request closed without merging**